src/App.vala

```diff
@@ -120,6 +122,27 @@ public class Dock.App : Object {
             pinned_action.set_state (pinned);
             LauncherManager.get_default ().sync_pinned ();
         });
+
+        var close_item = new GLib.MenuItem (_(CLOSE_WINDOWS_LABEL), ACTION_PREFIX + CLOSE_WINDOWS_ACTION);
```

Comment:
You shouldn't add `CLOSE_WINDOWS_LABEL`, it is untranslatable

Reply:
Well, one can if it is marked as N_,
So it has to be:
```vala
    private const string CLOSE_WINDOWS_LABEL = N_("Close All Windows");
```